crash_rate_aggregates/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+<<ENDCOMMENT
+# these setup routines are used to set up an environment when non is available, like in Vagrant
+# since the script usually runs on the Telemetry Spark Analysis environment, the requirements should already be installed
+
+# install packages
+sudo apt-get install -y libsnappy-dev liblzma-dev openjdk-7-jdk
+sudo pip2 install telemetry-tools
+sudo apt-get install -y python-numpy python-pandas # these are dependencies of python_moztelemetry, but installing them with apt-get massively speeds up python_moztelemetry installation
```

Comment:
Like in the ansible config above, this is also for speed, since installing these packages makes the conda install a lot faster as well.